src/spikeinterface/core/sorting_tools.py

```diff
@@ -0,0 +1,90 @@
+import numpy as np
+
+
+def spike_vector_to_spike_trains(spike_vector: list[np.array], unit_ids: np.array) -> dict[dict]:
+    """
+    Computes all spike trains for all units/segments from a spike vector list.
+
+    Internally call numba if numba is installed.
+
+    Parameters
+    ----------
+    spike_vector: list[np.ndarray]
+        List of spike vector optained with sorting.to_spike_vector(concatenated=False)
+    unit_ids: np.array
+        Unit ids
+
+    Returns
+    -------
+    spike_trains: dict[dict]:
```

Comment:
is this `list[dict]` or `dict[dict]`?
```suggestion
    spike_trains: list[dict]:
```